tensorflow/compiler/mlir/lite/transforms/prepare_composite_functions_tf.cc

```diff
@@ -225,7 +225,8 @@ void PrepareCompositeFunctionsPass::ConvertTFImplementsWithAttributes(
 LogicalResult CheckOutputConsumer(
     Operation* call_op, int expected_num_outputs,
     llvm::DenseSet<int> expected_consumer_indices) {
-  if (call_op->getNumResults() != expected_num_outputs) return failure();
+  const int call_op_getNumResults = call_op->getNumResults();
+  if (call_op_getNumResults != expected_num_outputs) return failure();
```

Comment:
```suggestion
  const int num_results = call_op->getNumResults();
  if (num_results != expected_num_outputs) return failure();
```

Much more readable name and follows style guide.